### DISCUSSION TITLE
Creating small community to evolve web VRM usecases

### DISCUSSION THREAD

**Discussion by witnesstodark** [pixiv/three-vrm#1150] [Showcase]
Hello to all devs! I hope everyone still alive in 23))

I and my team are working on virtual spaces project on the web and one core thing we implemented was supporting VRM models. 
I can't say it was easy, especially since animations were a pain. However, we did it.

First, I believe you would like to see our version of viewer - https://vrm-viewer.ownverse.world
Why it's cool and why it took half a year.
- We did GLB animation import... A lot of work there, but now animations is not a problem. We can convert mocaps, mixamo, basically everything
- Face sync based on ARkit. We attached Hallway technology there and it seems to work nice
- Adjusting spring bones in realtime. Very useful when your pipelines are aside of Unity and you are building web projects
- Combining parts.... (still in progress) That was a challenge but it must be very big advantage later. So basically we can create configurator based on VRM later. And physics will still work
- and more thing that may be common for you sir
![image](https://user-images.githubusercontent.com/65425917/210832404-e79f525f-0a03-4b1f-896e-4f724d61a1b6.png)

And, we actually tested it in virtual spaces - https://ownspace.ownverse.world
Must be looking nice, and remember, it's pure web, no Unity WebGL or something

So our idea was to build sort of community around technology cuz we like it a lot. What we do is commercial project lately, but we want to make core of it open source, as well as all stuff related to VRM etc. So if there are any enthusiasts, be my beloved quests in our discord (vrm-viewer channel) - https://discord.gg/wmAmmyxk

**Comment by tejasnayak25** [comment]
That's an amazing idea. I loved the ownspace and would love to know more about it. Will be looking forward to further updates.
But I noticed a few things. While moving in ownspace, there's no walking animation or running animation being played. Is it because of some error or because it's not set to work so?
But the map was a very stunning one.
Loved it.

**Reply by witnesstodark** [reply]
I think animations was just not loaded. Our app published on NL servers.. But we will check, ty for awesome feedback!
